packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -69,16 +69,12 @@ const PersonalDetailsForm = props => {
         if (
             !no_confirmation_needed &&
             typeof status === 'object' &&
-            !values.confirmation_checkbox &&
-            is_qualified_for_idv
+            !status.is_confirmed &&
```

Comment:
Aren't `values.confirmation_checkbox` and `status.is_confirmed` based on the same checkbox?

Reply:
I will refactor this flow in wall-1551 with new scrolling behaviour for forms